odxtools/compumethods/texttablecompumethod.py

```diff
@@ -29,10 +29,10 @@ def convert_physical_to_internal(self, physical_value):
 
     def __is_internal_in_scale(self, internal_value, scale: CompuScale):
         assert scale.lower_limit is not None
-        if scale.lower_limit.value is None:
+        if scale.lower_limit.value is not None:
```

Comment:
I agree that something is fishy here, but I think the new version is not correct either: first, the assert on line 31 ensures that the lower limit is not None (should be removed?) and probably the meat of the function is meant to be something like

```python
if scale.lower_limit is not None and not scale.lower_limit.complies_to_lower(internal_value):
    return False
if scale.upper_limit is not None and not scale.upper_limit.complies_to_upper(internal_value):
    return False

# value complies to the defined limits
return True
```

Reply:
Thanks for the catch. Our test PDX files used only the lower_limits. Updated the code